controllers/virtualmachinepublishrequest/virtualmachinepublishrequest_controller_intg_test.go

```diff
@@ -164,8 +164,9 @@ func virtualMachinePublishRequestReconcile() {
 
 					// Force trigger a reconcile.
 					vmPubObj := getVirtualMachinePublishRequest(ctx, client.ObjectKeyFromObject(vmpub))
+					beforeVMPubObj := vmPubObj.DeepCopy()
 					vmPubObj.Annotations = map[string]string{"dummy": "dummy-1"}
-					Expect(ctx.Client.Update(ctx, vmPubObj)).To(Succeed())
+					Expect(ctx.Client.Patch(ctx, vmPubObj, client.MergeFrom(beforeVMPubObj))).To(Succeed())
```

Comment:
Why not use https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/controller/controllerutil#CreateOrPatch? Same for below?

Reply:
Updated.